www/src/config.js

```diff
@@ -1,6 +1,6 @@
-import hasFeatureFlagEnabled from './utils/hasFeatureFlagEnabled';
+// import hasFeatureFlagEnabled from './utils/hasFeatureFlagEnabled';
```

Comment:
unused var discovered by eslint warning seen locally. not sure why not caught by CI without further investigation.